RocketChat/Rocket.Chat.ReactNative
# fix: enable video call ringer only for direct messages

## Proposed changes

Disable ringer on group calls, since we don't wait for other user accept the call.

## Issue(s)	


closes: #5424

## How to test or reproduce


- In any channel, tap to initiate a call

- Verify the Action Sheet opens

- Tap “Call”

- Verify that the user is redirected to the Conference Room

## Screenshots

## Types of changes




## Checklist



## Further comments


Jira: [VC-142]


[VC-142]: https://rocketchat.atlassian.net/browse/VC-142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
